src/setup.py

```diff
@@ -1,9 +1,11 @@
 from setuptools import setup
-import pandas as pd
+import json
+
+with open("./supy/supy_version.json") as f:
+    ser_ver = json.load(f)
```

Comment:
Please rename `see_ver` to `dict_ver` so the type can be explicitly reflected from the name, which is the coding convention in supy. Thanks!